### ISSUE TITLE
Variant Picker still not working, new error

### ISSUE THREAD

**Issue by robotswordfight** [mirumee/saleor#3136]
### What I'm trying to achieve
After trying the fix from #3127 or even starting from scratch all pages now have the following error:

```
Uncaught TypeError: Cannot read property 'isPrototypeOf' of undefined
    at J (storefront.js:144)
    at Module.641 (storefront.js:144)
    at n (storefront.js:1)
    at Object.1384 (storefront.js:15)
    at n (storefront.js:1)
    at Object.<anonymous> (storefront.js:15)
    at Object.1383 (storefront.js:15)
    at n (storefront.js:1)
    at Object.1368 (storefront.js:9)
    at n (storefront.js:1)
```

### Steps to reproduce the problem
1. Fresh install & Setup 618ad655ebdb3e5f2937e43855fe3e7cb9d23515
2. Variant picker is still broken, new error now (shown above) exists on all pages. 

Not sure what else may be broken.

**Comment by dominik-zeglen** [comment]
@robotswordfight we cannot reproduce the error you reported. Can you give us more info, like Node version?

**Comment by robotswordfight** [comment]
➜ node -v
v10.12.0

**Comment by robotswordfight** [comment]
Ah! It only happens with the development react.

I changed this:

```
  reactPath = 'node_modules/react/cjs/react.development.js';
  reactDomPath = 'node_modules/react-dom/cjs/react-dom.development.js';
```
to this:

```
  reactPath = 'node_modules/react/cjs/react.production.min.js';
  reactDomPath = 'node_modules/react-dom/cjs/react-dom.production.min.js';
```

and no more error and variant works!

**Comment by arlejeun** [comment]
Sorry but I'm facing similar issue with a fresh new install on Ubuntu 18.04.
Node version - 10.12.0

I am somewhat familiar with Django and python but I have no knowledge about react and very little with webpack. It looks like the VariantPicker is not properly constructed causing an issue in the console and no element to be display in my detail product page. 

See below the result in the console when I run the webserver and try to show the details of the product
http://10.0.0.151:8000/en/products/chambers-williams-4/
storefront.js:144 Uncaught TypeError: Cannot read property 'isPrototypeOf' of undefined
    at J (storefront.js:144)
    at Module.641 (storefront.js:144)
    at n (storefront.js:1)
    at Object.1384 (storefront.js:15)
    at n (storefront.js:1)
    at Object.<anonymous> (storefront.js:15)
    at Object.1383 (storefront.js:15)
    at n (storefront.js:1)
    at Object.1368 (storefront.js:9)
    at n (storefront.js:1)

**Expected behavior**
![expected screenshot - details page](https://user-images.githubusercontent.com/4977163/47322335-2ea8a500-d61d-11e8-9411-56ad0063d822.png)

**Observed result**
![screenshot - details page](https://user-images.githubusercontent.com/4977163/47322336-2ea8a500-d61d-11e8-92af-3be9d1dfbba1.png)

**Comment by dinhnghia** [comment]
> Sorry but I'm facing similar issue with a fresh new install on Ubuntu 18.04.
> Node version - 10.12.0
> 
> I am somewhat familiar with Django and python but I have no knowledge about react and very little with webpack. It looks like the VariantPicker is not properly constructed causing an issue in the console and no element to be display in my detail product page.
> 
> See below the result in the console when I run the webserver and try to show the details of the product
> http://10.0.0.151:8000/en/products/chambers-williams-4/
> storefront.js:144 Uncaught TypeError: Cannot read property 'isPrototypeOf' of undefined
> at J (storefront.js:144)
> at Module.641 (storefront.js:144)
> at n (storefront.js:1)
> at Object.1384 (storefront.js:15)
> at n (storefront.js:1)
> at Object. (storefront.js:15)
> at Object.1383 (storefront.js:15)
> at n (storefront.js:1)
> at Object.1368 (storefront.js:9)
> at n (storefront.js:1)
> 
> **Expected behavior**
> ![expected screenshot - details page](https://user-images.githubusercontent.com/4977163/47322335-2ea8a500-d61d-11e8-9411-56ad0063d822.png)
> 
> **Observed result**
> ![screenshot - details page](https://user-images.githubusercontent.com/4977163/47322336-2ea8a500-d61d-11e8-92af-3be9d1dfbba1.png)
> 
> Can you please advise?
> 
> Thanks,
> Arnaud

I have the same problem, have you fixed it yet?

**Comment by NyanKiyoshi** [comment]
> Ah! It only happens with the development react.
> 
> I changed this:
> 
> ```
>   reactPath = 'node_modules/react/cjs/react.development.js';
>   reactDomPath = 'node_modules/react-dom/cjs/react-dom.development.js';
> ```
> 
> to this:
> 
> ```
>   reactPath = 'node_modules/react/cjs/react.production.min.js';
>   reactDomPath = 'node_modules/react-dom/cjs/react-dom.production.min.js';
> ```
> 
> and no more error and variant works!

Did you try this as well?

**Comment by HenryMehta** [comment]
I've just installed a new version (latest pull of master) and I'm getting this issue with the example data.  The juices are not showing variants

**Comment by maarcingebala** [comment]
I think there may be something wrong with the example data from `populatedb_data.json`. We'll investigate the issue.

**Comment by maarcingebala** [comment]
This shuold be resolved by #3926.
